spec/cc/analyzer/source_fingerprint_spec.rb

```diff
@@ -61,6 +61,23 @@ module CC::Analyzer
 
         expect(fingerprint.compute).to eq("eef541a28f83417a45808139d58b631d")
       end
+
+      it "clears out invalid UTF-8 bytes" do
+        output["location"] = {
+          "lines" => {
+            "begin" => 1,
+            "end" => 1
+          },
+          "path" => "spec/fixtures/source_invalid_utf8.rb"
+        }
+
+        allow(File).to receive(:read).and_return("hi \255".force_encoding(Encoding::UTF_8))
```

Comment:
Ahh, that's right. The fixture file wasn't working so I switched tactics. I'll take out the fixture file.